pyiron/lammps/control.py

```diff
@@ -789,6 +789,65 @@ def energy_pot_per_atom(self):
             self['dump___1'] += ' c_energy_pot_per_atom'
             self['dump_modify___1'] = self['dump_modify___1'][:-1] + ' %20.15g"'
 
+    def _set_group_by_id(self, group_name, ids):
+        if len(ids) < 1:
+            raise ValueError('Group ids must have at least length one, but got {}'.format(ids))
+        if np.any([isinstance(id_, bool) or not isinstance(id_, (int, np.int64)) for id_ in ids]):
```

Comment:
```suggestion
        if np.any([isinstance(id_, bool) or not isinstance(id_, (int, np.integer)) for id_ in ids]):
```

Reply:
*sigh* right, so the windows systems are not 64 bit or something silly like this. Ugh. But thank you! I don't think I ever would have thought down this line myself!

Reply:
Programming for windows is annoying - we had the same issue before https://github.com/pyiron/pyiron/blob/master/pyiron/atomistics/structure/sparse_list.py#L174 and https://github.com/pyiron/pyiron/blob/master/pyiron/atomistics/structure/atoms.py#L136

Reply:
Maybe it makes sense to have a function in `pyiron_base` which provides this kind of utility tasks like https://github.com/pyiron/pyiron_base/blob/master/pyiron_base/generic/util.py#L37